src/components/CompanyFilterButton/index.tsx

```diff
@@ -174,12 +263,30 @@ export function CompanyFilterButton(props: CompanyFilterButtonProps): ReactEleme
         pause: !state.searchTerm,
     });
 
+    // Keep option in view when scrolling by keyboard
+    useEffect(() => {
```

Comment:
`useAutoScroll` _should_ automatically do this for you. Would love to use that here too if possible, and if there's issues then I should probably fix/add whatever is needed to make it work.